src/rgw/driver/rados/rgw_pubsub_push.cc

```diff
@@ -122,7 +122,6 @@ namespace {
 class Waiter {
   using Signature = void(boost::system::error_code);
```

Comment:
I should probably make an attempt to get rid of this class entirely, but that's not urgent.

Reply:
currently, it is only used to yield a coroutine.
so, for starter, we can remove the mutex+cond var code.